x/nftmarket/keeper/nft_listing.go

```diff
@@ -223,7 +276,7 @@ func (k Keeper) ListNft(ctx sdk.Context, msg *types.MsgListNft) error {
 		StartedAt:     ctx.BlockTime(),
 		EndAt:         ctx.BlockTime().Add(time.Second * time.Duration(params.NftListingPeriodInitial)),
 	}
-	k.SetNftListing(ctx, listing)
+	k.SaveNftListing(ctx, listing)
```

Comment:
Is there any specific reason to change function name from SetNftListing to SaveNftListing? Usually, for low level operations, Set prefix is used for common Cosmos SDK modules.

Reply:
SaveNftListing calls two function.
it's SetNftListing and SetListedClass.
I did not want to call SetListedClass within SetNftListing.
What should I do?

Reply:
Okay got it - I thought it's renaming of function